stable/redis-ha/templates/tests/test-redis-ha-configmap.yaml

```diff
@@ -13,6 +15,8 @@ spec:
     args:
     - --shell=sh
     - /readonly-config/init.sh
+    securityContext:
```

Comment:
I think it would be safe to template this based off `.Values.securityContext`

Reply:
right, done that